src/server/helpers/config.js

```diff
@@ -16,8 +16,19 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+import {readFileSync} from 'fs';
+
+
 /* Pull in environment-specific configuration. */
-const env = process.env.NODE_ENV || 'development';
-const config = require(`../../../config/${env}`);
+const env = process.env.NODE_ENV || 'development'; // eslint-disable-line no-process-env,max-len
+
+let configFileBasename = 'config';
+if (env === 'test') {
+	configFileBasename = 'test';
+}
+
+const configContents =
```

Comment:
There must be a way that we can do this better... Some sort of asynchronous caching config loader module...

Something that we can use in places where config is consumed, like this:

    const config = await new Config();
    initLibrary(config.data.here);